cli/main.rs

```diff
@@ -1181,9 +1202,22 @@ fn get_subcommand(
     DenoSubcommand::Lint {
       files,
       rules,
+      rules_tags,
+      rules_include,
+      rules_exclude,
       ignore,
       json,
-    } => lint_command(flags, files, rules, ignore, json).boxed_local(),
+    } => lint_command(
+      flags,
+      files,
+      rules,
+      rules_tags,
+      rules_include,
+      rules_exclude,
+      ignore,
+      json,
+    )
+    .boxed_local(),
```

Comment:
These commands are getting really bad... we've got a few subcommands that have more than 5 arguments.

I think it might be high time to refactor `DenoSubcommand` enum to something like:
```rust
#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
LintSubcommand {
  files: Vec<PathBuf>,
  ignore: Vec<PathBuf>,
  rules: bool,
  rules_tags: Vec<String>,
  rules_include: Vec<String>,
  rules_exclude: Vec<String>,
  json: bool,
}

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub enum DenoSubcommand {
  ...
  Lint(LintSubcommand),
  ...
}

```